### ISSUE TITLE
Can't get R scripts to work (cannot find lineupOutput function)

### ISSUE THREAD

**Issue by mrpsonglao** [asRodelgo/TCdata360_playground#1]
`Error in lineupOutput("lineup1") : could not find function "lineupOutput"`

**Comment by asRodelgo** [comment]
Reg, you need to install lineup package from github (see htmlwidgets gallery). That function should come within. Let me know.

**Comment by mrpsonglao** [comment]
Yeah, I've already installed the lineup package from GitHub when I got that
error. Any other possibilities you have in mind?

**Comment by asRodelgo** [comment]
You need to run it calling shiny package as well:
library(shiny). That function is shiny specific.
